<launch>
  <node name="controller" pkg="multi_turtles" type="PID_controller.py" output="screen">
    <param name="kp_ang" value="30"/>
    <param name="ki_ang" value="0"/>
    <param name="kd_ang" value="15"/>
    <param name="kp_lin" value="0.6"/>
    <param name="ki_lin" value="0.01"/>
    <param name="kd_lin" value="6"/>
  </node>
 
  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_gazebo_rviz.launch"/>
  <node name="monitor" pkg="multi_turtles" type="monitor.py" output="screen"/>

</launch>